Tests/MapboxMapsTests/Foundation/Integration Tests/Map/MapViewRenderedSnapshotIntegrationTests.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+import XCTest
+@testable @_spi(Experimental) import MapboxMaps
+
+final class MapViewRenderedSnapshotIntegrationTests: MapViewIntegrationTestCase {
+
+    func testLoadStyleAndTakeSnapshotSucceeds() {
+
+        guard let style = style else {
+            XCTFail("Should have a valid Style object")
+            return
+        }
+
+        let expectation1 = self.expectation(description: "Wait for style to load")
```

Comment:
let's use more descriptive names for these expectations.

This one can be `waitForStyleExpectation`